pkg/cloud/types.go

```diff
@@ -757,20 +767,6 @@ type CloudAwsGovcloudIntegrationsInput struct {
 	Sqs []CloudSqsIntegrationInput `json:"sqs,omitempty"`
 }
 
-// CloudAwsGovcloudLinkAccountInput - Information required to link an AWS GovCloud account to a NewRelic account.
-type CloudAwsGovcloudLinkAccountInput struct {
-	// The key used to make requests to AWS service APIs
-	AccessKeyId string `json:"accessKeyId"`
-	// The AWS account id
-	AwsAccountId string `json:"awsAccountId"`
-	// How metrics will be collected.
-	MetricCollectionMode CloudMetricCollectionMode `json:"metricCollectionMode,omitempty"`
-	// The linked account name.
-	Name string `json:"name"`
-	// The secret key used to make requests to AWS service APIs
-	SecretAccessKey SecureValue `json:"secretAccessKey"`
-}
-
```

Comment:
A note for the reviewer: changes show this type to have been deleted, as it has been replaced by ```CloudAwsGov**C**loudLinkAccountInput``` (since the **C** in 'Govcloud' was made a capital letter in the NerdGraph API, we see a change in the type via Tutone). Types with similar names also show similar changes in this file.